git_machete/cmd.py

```diff
@@ -3076,6 +3187,38 @@ def usage(c: str = None) -> None:
             Please see hook_samples/ directory of git-machete project for examples.
             An example of using the standard git `post-commit` hook to `git machete add` branches automatically is also included.
         """,
+        "hub": """
+            <b>Usage: git machete hub <subcommand></b>
+            where <subcommand> is one of: `anno-prs`, `create-pr`, `retarget-pr`.
+
+            Creates, checks out and manages GitHub PRs while keeping them reflected in branch definition file.
+
+            For all subcommands, to allow GitHub API access for private repositories
+            (and also to correctly identify the current user, even in case of public repositories),
+            `GITHUB_TOKEN` env var must contain a GitHub API token with `repo` scope, see `https://github.com/settings/tokens`.
+
+            <b>`anno-prs`:</b>
+
+              Annotates the branches based on their corresponding GitHub PR numbers and authors.
+              Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
+              Equivalent to `git machete anno --sync-github-prs`.
+
+            <b>`create-pr [--draft]`:</b>
+
+              Creates a PR for the current branch, using the upstream (parent) branch as the PR base.
+              Once the PR is successfully created, annotates the current branch with the new PR's number.
+
+              If `.git/info/description` file is present, its contents is used as PR description.
+              If `.git/info/milestone` file is present, its contents (a single number - milestone id) is used as milestone.
+              If `.git/info/reviewers` file is present, its contents (one GitHub login per line) are used to set reviewers.
+
+              <b>Options:</b>
+                <b>--draft</b>    Creates the new PR as a draft.
+
```

Comment:
Also planned: `checkout-pr` (or whatever new name it should assume to be more `gh`-compatible), but this will be significantly more complex since it ideally should be able to traverse the PR chain upwards, adding parents to the branch layout one by one

Reply:
Added as https://github.com/VirtusLab/git-machete/issues/175